axlearn/audio/asr_decoder_test.py

```diff
@@ -89,6 +91,75 @@ def test_map_label_sequences(
         )
 
 
+class ValidCtcSeqTest(TestCase):
+    def get_logits_and_labels(self, batchsize, timesteps, labelsteps, nclasses):
+        logits = np.random.randn(batchsize, timesteps, nclasses)
+        logitpaddings = np.zeros((batchsize, timesteps), dtype=np.int32)
+        labels = np.random.randint(1, nclasses, size=(batchsize, labelsteps)).astype(np.int32)
+        labelpaddings = np.zeros((batchsize, labelsteps), dtype=np.int32)
+        return logits, logitpaddings, labels, labelpaddings
+
+    def test_label_longer_than_input(self):
+        batchsize = 4
+        timesteps = 10
+        labelsteps = 11
+        nclasses = 400
+        # generate logits and labels, which has logits shorter than labels
+        logits, logitpaddings, labels, labelpaddings = self.get_logits_and_labels(
+            batchsize, timesteps, labelsteps, nclasses
+        )
+        per_seq_loss = optax.ctc_loss(logits, logitpaddings, labels, labelpaddings, blank_id=0)
+        print(per_seq_loss)
```

Comment:
nit: Remove debugging prints here and elsewhere?

Reply:
Here, I try to show that when presnets an invalid sequence, `optax.ctc_loss` returns somethong abnormal. On the other hand, I don't want to assert this loss value, as it can change anytime. What's the best strategy here ? Maybe I can using logging.info instead ?

Reply:
would assert_greater() works?